### PR TITLE
new icon: jetbrains (plain, original)

### PR DESCRIPTION
**Pull request by tupaschoal**
`Original`
jetbrains-original
`Plain`
jetbrains-plain

I removed the eps, as it's not applicable anymore. I also removed the aliases, as the new icon doesn't look like a line.

### CODE REVIEW

File: icons/jetbrains/jetbrains-plain.svg [hunk 1]
```diff
@@ -1 +1,2 @@
...
+
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:2]:
```suggestion
```

File: icons/jetbrains/jetbrains-original.svg [hunk 2]
```diff
@@ -1 +1,2 @@
...
+
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-original.svg:2]:
```suggestion
```

File: devicon.json [hunk 3]
```diff
@@ -2039,27 +2039,11 @@
...
-            {
-                "base": "plain",
-                "alias": "line-wordmark"
-            },
-            {
-                "base": "plain",
-                "alias": "plain-wordmark"
-            }
-        ]
+        "color": "#000000",
```
Comment (Panquesito7) [inline devicon.json:2045]: I recommend this color instead: `#FDCC21`
```suggestion
        "color": "#FDCC21",
```

File: devicon.json [hunk 4]
```diff
@@ -2039,27 +2039,11 @@
                 "plain"
```
Comment (Panquesito7) [inline devicon.json:2039]: In the line 2035, do you think you could add some tags?

**Review by Panquesito7 — COMMENTED** [review]
I'm not sure if removing the `.eps` file is a good idea. @Thomas-Boi, thoughts?

File: icons/jetbrains/jetbrains-plain.svg [hunk 5]
```diff
@@ -1 +1,2 @@
...
....016-.024c0-.367-.3-.566-.84-.566zm6.324.113v1.614l.992-.016c.621 0 .996-.305.996-.793 0-.535-.39-.805-1-.805zm7.3.54l-.753 1.886h1.5zM49.18 61.25v1.227h1.152c.57 0 .883-.22.883-.61-.016-.379-.285-.617-.903-.617zm-1.871 16.082h16V80h-16zm0 0"/><path d="M63.895 18.438A7.747 7.747 0 0051.91 11.91L16 33.714a10.66 10.66 0 00-4.793 9.96 10.67 10.67 0 0017.523 7.106l32.192-26.234.254-.211a7.783 7.783 0 002.719-5.898zm0 0"/><path d="M116.117 65.422L61.633 12.949a7.763 7.763 0 00-7.692-1.965 7.763 7.763 0 00-5.394 5.825 7.764 7.764 0 002.555 7.52l.097.085 57.887 48.766a5.235 5.235 0 005.578.734 5.243 5.243 0 003.02-4.75 5.281 5.281 0 00-1.567-3.742zm0 0"/><path d="M117.688 69.184a5.226 5.226 0 00-8.297-4.266l-65.926 32.21a10.668 10.668 0 1011.008 18.2l60.96-41.844a5.21 5.21 0 002.254-4.3zm0 0"/><path d="M75.64 62.953l1.024-1.23a3.45 3.45 0 002.133.804c.508 0 .77-.175.77-.46 0-.294-.215-.434-1.13-.649-1.425-.328-2.511-.711-2.511-2.082 0-1.238.98-2.133 2.582-2.133a4.185 4.185 0 012.738.887l-.89 1.265a3.244 3.244 0 00-1.864-.652c-.449 0-.668.176-.668.426 0 .305.227.441 1.16.648 1.528.332 2.473.832 2.473 2.075 0 1.359-1.074 2.132-2.687 2.132a4.643 4.643 0 01-3.13-1.03zm0 0" fill="#fff"/></svg>
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M21.715 32.152a10.663 10.663 0 00-10.492 8.938 10.664 10.664 0 007.082 11.824h.105l.277.086 23.981 7.305v25.027h42.664V42.668H42.668v3.883L28.594 34.64a10.65 10.65 0 00-6.88-2.489zm27.914 15.864h1.832v4.222a2.434 2.434 0 01-.652 1.871 2.528 2.528 0 01-1.829.711 2.745 2.745 0 01-2.34-1.152l1.188-1.125c.23.363.621.594 1.047.621.457 0 .754-.32.754-.941zm2.66 0h5.356v1.554h-3.56v1.02h3.196v1.422h-3.164v1.058h3.555v1.563h-5.406zm5.973 0h5.84v1.613h-2v5.043h-1.848v-5.043h-1.992zm3.668 9.234h1.777l2.848 6.715H64.57l-.476-1.203h-2.582L61.035 64H57.45l-1.344-2.02h-.617V64h-1.847l.011-6.7h2.953c.098-.007.196-.01.293-.007a2.926 2.926 0 011.875.719c.403.414.618.972.59 1.55a2.132 2.132 0 01-1.37 2.055l1.296 1.946zm4.914.031h1.847v6.66h-1.847zm2.531 0h1.734l2.399 3.555V57.28h1.828v6.66h-1.613L71.21 60.29v3.652h-1.836zm-18.605.012a2.47 2.47 0 011.566.598c.289.285.445.675.437 1.082a1.556 1.556 0 01-1.03 1.484c.78.145 1.339.836 1.32 1.629 0 1.203-.93 1.914-2.524 1.914h-3.156v-6.7h3.14c.082-.007.164-.007.247-.007zm-1.59 1.496v1.164h.89c.555 0 .883-.191.883-.574l-.016-.024c0-.367-.3-.566-.84-.566zm6.324.113v1.614l.992-.016c.621 0 .996-.305.996-.793 0-.535-.39-.805-1-.805zm7.3.54l-.753 1.886h1.5zM49.18 61.25v1.227h1.152c.57 0 .883-.22.883-.61-.016-.379-.285-.617-.903-.617zm-1.871 16.082h16V80h-16zm0 0"/><path d="M63.895 18.438A7.747 7.747 0 0051.91 11.91L16 33.714a10.66 10.66 0 00-4.793 9.96 10.67 10.67 0 0017.523 7.106l32.192-26.234.254-.211a7.783 7.783 0 002.719-5.898zm0 0"/><path d="M116.117 65.422L61.633 12.949a7.763 7.763 0 00-7.692-1.965 7.763 7.763 0 00-5.394 5.825 7.764 7.764 0 002.555 7.52l.097.085 57.887 48.766a5.235 5.235 0 005.578.734 5.243 5.243 0 003.02-4.75 5.281 5.281 0 00-1.567-3.742zm0 0"/><path d="M117.688 69.184a5.226 5.226 0 00-8.297-4.266l-65.926 32.21a10.668 10.668 0 1011.008 18.2l60.96-41.844a5.21 5.21 0 002.254-4.3zm0 0"/><path d="M75.64 62.953l1.024-1.23a3.45 3.45 0 002.133.804c.508 0 .77-.175.77-.46 0-.294-.215-.434-1.13-.649-1.425-.328-2.511-.711-2.511-2.082 0-1.238.98-2.133 2.582-2.133a4.185 4.185 0 012.738.887l-.89 1.265a3.244 3.244 0 00-1.864-.652c-.449 0-.668.176-.668.426 0 .305.227.441 1.16.648 1.528.332 2.473.832 2.473 2.075 0 1.359-1.074 2.132-2.687 2.132a4.643 4.643 0 01-3.13-1.03zm0 0"/></svg>
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Thomas-Boi** [comment]
Hey @Panquesito7,

Since he's updating Jetbrains, he's removing the old `.eps` file. I think this is fine since it doesn't make sense to have an old `.eps` file while the icons are different

File: devicon.json (hunk 4 above)
Comment (tupaschoal) [reply devicon.json:2062]: Well, it's a company that does lots of IDEs, so I added `ide`, couldn't think of anything else.

**Comment by Thomas-Boi** [comment]
Another great icon @tupaschoal ! Thank you for your work 😄

**Review by Thomas-Boi — APPROVED** [review]

**Comment by tupaschoal** [comment]
I'm enjoying doing it :D

**Pull request merged**